test.sh

```diff
@@ -302,4 +303,10 @@ if [[ "$RUN" =~ "godep-restore" ]] ; then
   end_context #godep-restore
 fi
 
+if [[ "$RUN" =~ "errcheck" ]] ; then
+  start_context "errcheck"
+  run_and_comment errcheck -ignore 'io:Write,Close,os:Remove,github.com/letsencrypt/boulder/Godeps/_workspace/src/gopkg.in/gorp.v1:Rollback,net/http:Write' -ignorepkg 'github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cactus/go-statsd-client/statsd,github.com/letsencrypt/boulder/log,github.com/letsencrypt/boulder/metrics,github.com/letsencrypt/boulder/rpc/pb' ./akamai/ ./bdns/ ./bin/ ./ca/ ./cmd/ ./core/ ./data/ ./log/ ./mail ./metrics ./mocks ./policy ./probs ./publisher ./ra ./reloader ./rpc ./sa ./test ./va ./wfe
```

Comment:
The package list was to ignore calls to the packages, like auditlogger.Warning(), but yeah I can move those.

The fixes to the Rollback calls were before I saw how many Rollback calls were in the SA, and then I looked up under which circumstances exactly mysql will fail a rollback. Basically, if the transaction was already committed.